kura/org.eclipse.kura.rest.packages.provider/OSGI-INF/deployment_rest_service.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Eurotech and/or its affiliates and others
+
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+	SPDX-License-Identifier: EPL-2.0
+
+	Contributors:
+	 Eurotech
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.kura.rest.deployment.agent.DeploymentRestService">
+   <implementation class="org.eclipse.kura.internal.rest.deployment.agent.DeploymentRestService"/>
+   <property name="kura.service.pid" type="String" value="org.eclipse.kura.rest.deployment.agent.DeploymentRestService"/>
+   <service>
+      <provide interface="org.eclipse.kura.internal.rest.deployment.agent.DeploymentRestService"/>
+   </service>
+   <reference bind="setDeploymentAgentService" cardinality="1..1" interface="org.eclipse.kura.deployment.agent.DeploymentAgentService" name="DeploymentAgentService" policy="static" unbind="unsetDeploymentAgentService"/>
```

Comment:
Typically if the policy is static, the unbind method is not required.

Reply:
Fixed in fd6188a036de2fdbe67b4cb2c5d89f374edb26f8